operations/patch_module.go

```diff
@@ -27,7 +28,6 @@ func PatchSetModule() cli.Command {
 		Before: mergeBeforeFuncs(
 			autoUpdateCLI,
 			setPlainLogger,
-			requirePatchIDFlag,
```

Comment:
Can we add text to the patch ID flag usage to say that we default to the latest submitted patch?

Reply:
added it manually but if there's a better way to do this that preserves the order let me know!